pkg/klog/default_test.go

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2021 CloudWeGo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package klog
+
+import (
+	"bytes"
+	"context"
+	"os"
+	"testing"
+
+	"github.com/cloudwego/kitex/internal/test"
+)
+
+// test package level functions without format
+func normalOutput(t *testing.T, testLevel Level, want string, args ...interface{}) {
+	buf := new(bytes.Buffer)
+	SetOutput(buf)
+	defer SetOutput(os.Stderr)
+	switch testLevel {
+	case LevelTrace:
+		Trace(args...)
+		test.Assert(t, buf.String() == want)
+	case LevelDebug:
+		Debug(args...)
+		test.Assert(t, buf.String() == want)
+	case LevelInfo:
+		Info(args...)
+		test.Assert(t, buf.String() == want)
+	case LevelNotice:
+		Notice(args...)
+		test.Assert(t, buf.String() == want)
+	case LevelWarn:
+		Warn(args...)
+		test.Assert(t, buf.String() == want)
+	case LevelError:
+		Error(args...)
+		test.Assert(t, buf.String() == want)
+	case LevelFatal:
+		t.Fatal("fatal method cannot be tested")
+	default:
+		t.Errorf("unknow level: %d", testLevel)
+	}
+}
+
+// test package level Ctx-related functions with 'format'
+func ctxOutput(t *testing.T, testLevel Level, want, format string, args ...interface{}) {
+	buf := new(bytes.Buffer)
+	SetOutput(buf)
+	defer SetOutput(os.Stderr)
+
+	// TODO: fill ctx and complete Ctx-related methods
```

Comment:
Comment chang to 'the default logger implementation of CtxLogger is same as FormatLogger, no context handle now'